mlx/compile.cpp

```diff
@@ -266,6 +266,10 @@ class CompilerCache {
     cache_.erase(fun_id);
   }
 
+  std::unordered_map<std::uintptr_t, std::vector<CacheEntry>>& entries() {
+    return cache_;
+  }
```

Comment:
Do you think it makes sense to make this a `clear` function instead?

Reply:
Same as above initially added because I had more functionality for testing and didn't limit enough the API in detail. Will change it to clear.